### PR TITLE
OIE - add support for new pipeline in signin widget

### PR DESCRIPTION
**Pull request by haishengwu-okta**

### CODE REVIEW

File: test/unit/spec/RefreshAuthState_spec.js [hunk 1]
```diff
@@ -36,18 +36,21 @@ function (Q, Okta, OktaAuth, Util, Beacon, FormView, Expect,
...
+        ac: authClient,
+        setNextResponse: setNextResponse
+      });
     });
   }
 
   Expect.describe('RefreshAuthState', function () {
 
-    itp('redirects to PrimaryAuth if authClient does not need a refresh', function () {
+    /*itp('redirects to PrimaryAuth if authClient does not need a refresh', function () {
```
Comment (haishengwu-okta) [inline test/unit/spec/RefreshAuthState_spec.js]: @nikhilvenkatraman-okta did we forget to fix the test?

Comment (nikhilvenkatraman-okta) [reply test/unit/spec/RefreshAuthState_spec.js]: ah, looks like we missed to update this test.

Comment (nikhilvenkatraman-okta) [reply test/unit/spec/RefreshAuthState_spec.js]: Fixed [here](https://github.com/okta/okta-signin-widget/pull/899)

**Comment by haishengwu-okta** [comment]
Found we missed couple of commits during the squash.

**Pull request merged**